docs/content/config/advanced/override-defaults/dovecot.md

```diff
@@ -30,15 +30,20 @@ This limit is quickly reached if users connect to the mail server with multiple
 
 ## Override Configuration
 
-For major configuration changes it’s best to override the dovecot configuration files. For each configuration file you want to override, add a list entry under the `volumes` key.
-
-```yaml
-services:
-  mail:
-    volumes:
-      - maildata:/var/mail
-      - ./config/dovecot/10-master.conf:/etc/dovecot/conf.d/10-master.conf
-```
+For major configuration changes it’s best to override the dovecot configuration files. If you do not yet have a full config files populated on the host, it's unfortunately not as easy as mounting an empty host location to `/etc/dovecot/conf.d` on the container as the config files in the container will be deleted with an empty folder. Fortunately, there is a multi-step process necessary to obtain the current configs:
+
+1. Add a list entry under the `volumes` key for a temporary/empty folder:
+    ```yaml
+    services:
+      mail:
+        volumes:
+          - maildata:/var/mail
+          - ./config/dovecot:/etc/dovecot/conf.d-tmp
+    ```
+2. You can then start the container with `docker-compose up`
+3. Once started, execute `docker exec -it {id of container} bash -c "cp -rfp /etc/dovecot/conf.d/* /etc/dovecot/conf.d-tmp/"` and then `docker-compose down`.
```

Comment:
Instead of `{id of container}`, `mailserver` could be used, as this is our default container name.

Reply:
I don't use that as the name; I think it's more explicit for newbies this way.

Reply:
It's not, since this project and its documentation builds on the premise that the container is called `mailserver`. This is found everywhere in the documentation, and it should be kept this way.

Reply:
@aendeavor , should I then update the rest of the documentation? https://github.com/docker-mailserver/docker-mailserver/blob/8ce3cca4244ed2adbf936571d62e2ee43ca8d68a/docs/content/config/advanced/override-defaults/dovecot.md#debugging

I'd rather not if it can be avoided

Reply:
> @aendeavor , should I then update the rest of the documentation?
> I'd rather not if it can be avoided

No, this is something we as maintainers have to fix later:) But you will need to do it for the documentationyou introduced.